salesforce/10.19/modules/ROOT/pages/salesforce-connector-processing-events.adoc

```diff
@@ -208,12 +290,14 @@ In the following XML example, the *Replay topic listener* operation (`replay-top
 </flow>
 ----
 
-== Work with Streaming Channel Events
+== Working with Generic Events (Legacy)
+
+Work with generic events by creating a streaming channel, subscribing to a streaming channel, replaying generic events, and pushing events to a streaming channel.
 
-Work with streaming channel events by creating a streaming channel, subscribing to a streaming channel, replaying streaming channel events, and pushing events to a streaming channel.
+NOTE: Generic Events is a legacy product. Salesforce no longer enhances Generic Events with new features and provides limited support for this product. Instead of Generic Events, you can use Platform Events. To learn more about Platform Events, refer to the https://developer.salesforce.com/docs/atlas.en-us.246.0.platform_events.meta/platform_events/platform_events_intro.htm[Platform Events Developer Guide] and the https://trailhead.salesforce.com/content/learn/modules/platform_events_basics[Platform Events Basics] Trailhead module.
```

Comment:
```suggestion
[NOTE] 
====
Generic Events is a legacy product. Salesforce no longer enhances Generic Events with new features and provides limited support for this product. Instead of Generic Events, use Platform Events. To learn more about Platform Events, refer to the https://developer.salesforce.com/docs/atlas.en-us.246.0.platform_events.meta/platform_events/platform_events_intro.htm[Platform Events Developer Guide^] and the https://trailhead.salesforce.com/content/learn/modules/platform_events_basics[Platform Events Basics^] Trailhead module.
====
```